src/core/index.js

```diff
@@ -19,7 +19,8 @@ var core = module.exports = Object.assign(require('./const'), require('./math'),
     Container:              require('./display/Container'),
     Transform:              require('./display/Transform'),
     TransformStatic:        require('./display/TransformStatic'),
-    TransformManual:        require('./display/TransformManual'),
+    TransformBase:          require('./display/TransformBase'),
+    TransformManual:        require('./display/TransformBase'),
```

Comment:
Its v4-rc2 thing, it is used only in pixi-spine, we dont have to deprecate it, we'll just remove it before release of v4. I just dont want pixi-spine to be broken for a few days.